tox.ini

```
@@ -1,13 +1,16 @@
 [tox]
-envlist = py27,flake8
+envlist = py27-django111,py3-django2,flake8
 skipsdist = True
 
 [testenv]
-deps = -r{toxinidir}/requirements/test.txt
-       Django<2
+deps =
+    django111: Django<2
+    django22: Django<3
```

More explicit requirements.

```suggestion
    django111: Django>=1.11,<2.0
    django22: Django>=2.0,<2.3
```